core/tests/protractor_desktop/collections.js

```diff
@@ -156,12 +162,16 @@ describe('Collections', function() {
     collectionEditorPage.setObjective('This is a test collection.');
     collectionEditorPage.setCategory('Algebra');
     collectionEditorPage.saveChanges();
+    waitFor.invisibilityOf(
+      saveInProgressLabel, 'Collection is taking too long to save.');
```

Comment:
move this inside `collectionEditorPage.saveChanges` function.

Reply:
Done.